Menu buttons

JKadhim opened the pull request:
Removed the exit game buttons and added functinality to the main menu buttons. I cant currently test the multiplayer button

kieranknowles1 commented:
Your description isn’t entirely clear. Do you mean “remove exit game button from end screen”? And what do you mean by adding functionality to the main menu buttons?

JKadhim commented:
My bad. I added functionality to the return to main menu buttons on the end game screen and i removed the option to quit the game from the end game screen

A-Barnett commented:
Looks functional to me, we did not need quit buttons there if it goes direct to menu anyway you are correct. I added back in the check for deathcount as I believe Billy will use this for singleplayer lives
